doc/releases/changelog-0.21.0.md

```diff
@@ -379,10 +395,10 @@
   to control the precision of parameters.
   [(#2071)](https://github.com/PennyLaneAI/pennylane/pull/2071)
 
-* Insert transform now supports adding operation after or before certain specific gates.
+* `insert` transform now supports adding operation after or before certain specific gates.
   [(#1980)](https://github.com/PennyLaneAI/pennylane/pull/1980)
 
-* Interferometer is now a class with `shape` method.
+* Interferometer now has a `shape` method.
   [(#1946)](https://github.com/PennyLaneAI/pennylane/pull/1946)
 
 * The `CircuitGraph`, used to represent circuits via directed acyclic graphs, now
```

Comment:
@antalszava I can't comment on it, but the first entry in the Breaking Changes section says "See the previous entry for details.". I think it should be moved to the second entry!

Reply:
For the breaking changes section, let's have the following order:

1. QNode arguments will no longer be considered trainable by default when using the Autograd interface. 

2. qml.jacobian now follows a different convention regarding its output shape. [(#2059)](https://github.com/PennyLaneAI/pennylane/pull/2059)

3. qml.metric_tensor, qml.adjoint_metric_tensor and qml.transforms.classical_jacobian now follow a different convention regarding their output shape when being used with the Autograd interface [(#2059)](https://github.com/PennyLaneAI/pennylane/pull/2059)

4. The behaviour of RotosolveOptimizer has been changed regarding its keyword arguments. [(#2081)](https://github.com/PennyLaneAI/pennylane/pull/2081)

Reply:
Changed, thanks!